pkg/compose/v1/config.go

```diff
@@ -110,6 +113,32 @@ func NewDefaultConfig(options ...ConfigOpt) (*compose.Config, error) {
 		return nil, fmt.Errorf("failed to get file system stat; path: %s, err: %s", opts.StoreRoot, err.Error())
 	}
 
+	var proxyURL *url.URL
+	var proxyCerts *x509.CertPool
+	proxyToken := ""
+
+	proxyEnv := os.Getenv("COMPOSE_APPS_PROXY")
```

Comment:
Another not important at this stage nitpick: move the proxy related code into a dedicated function.